generators/server/templates/src/main/docker/config/realm-config/jhipster-realm.json.ejs

```diff
@@ -859,8 +859,7 @@
         "offline_access",
         "microprofile-jwt"
       ]
-    },
-<%_ if (serviceDiscoveryEureka) { _%>
+    },<% if (serviceDiscoveryEureka) { %>
```

Comment:
Why?

Reply:
To match the other file with the same content.